src/@batch-flask/ui/buttons/refresh-btn/refresh-btn.component.ts

```diff
@@ -59,6 +59,7 @@ export class RefreshButtonComponent implements OnDestroy {
     @autobind()
     public onClick() {
         this.status = RefreshStatus.Refreshing;
+        this.liveAnnouncer.announce("Refreshing in progress");
```

Comment:
Looks great, but can you make this a localized string? OK to also file a different issue to have that done at a later date.